### PR TITLE
OSDOCS-10290: machine mgmt for Alicloud removal 4.16 RN updates

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.16

Issue:
[OSDOCS-10290](https://issues.redhat.com//browse/OSDOCS-10290)

Link to docs preview:
* [Deprecated and removed features](https://80970--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-deprecated-removed-features_release-notes) (Machine management deprecated and removed features)
* [Managing machine with Machine API for Alibaba Cloud](https://80970--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-removed-features-alicloud-machine-management_release-notes)
* [Technology Preview features status](https://80970--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-technology-preview-tables_release-notes) (Machine management Technology Preview features)

QE review:
- [x] QE has approved this change.

Change management acks:
- [x] PX
- [x] QA engg
- [x] Dev engg
- [x] PM
- [x] Docs

Additional information:
Missed in 4.16, requires change mgmt
Content removal in #80824 
4.17 RN update in #80829

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3137,7 +3163,7 @@ In the following tables, features are marked with the following statuses:
 |Cloud controller manager for {alibaba}
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|Removed
```
Comment (huali9) [inline release_notes/ocp-4-16-release-notes.adoc:3166]: Same with https://github.com/openshift/openshift-docs/pull/80829#discussion_r1734096067

Comment (jeana-redhat) [reply release_notes/ocp-4-16-release-notes.adoc:3166]: see https://github.com/openshift/openshift-docs/pull/80829#discussion_r1734732263

Thanks for bringing this up! It highlighted a discrepancy in the RN tables that we will fix as a result :nerd_face:

Comment (huali9) [reply release_notes/ocp-4-16-release-notes.adoc:3166]: Good to know, thank you Jeana!
/lgtm

**Comment by kalexand-rh** [comment]
LGTM

**Comment by julienlim** [comment]
lgtm

**Comment by xltian** [comment]
lgtm

**Comment by jeana-redhat** [comment]
Dev engg lead acked via Slack DM :heavy_check_mark:

**Review by jldohmann — APPROVED** [review]
nice work Jeana! LGTM

**Pull request merged**